### PR TITLE
(zh-CN)Finish translation and markdownize learn/css/howto/css_faq

### PR DESCRIPTION
**Pull request by jasonren0403**
This PR has done:
* Translation of css_faq with newest docs
* Transfer html contents to markdown format

### CODE REVIEW

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+original_slug: Web/CSS/Common_CSS_Questions
+---
+## 为什么有效的 CSS 没有正确的渲染？
+
+浏览器使用 DOCTYPE 声明来选择是否使用更符合 Web 标准或兼容旧浏览器的 bug 的模式。在你的 HTML 的开始使用一个正确的和现代的 DOCTYPE 声明将改善浏览器标准执行。
+
+现代浏览器主要有两种渲染模式：
+
+- _怪异模式：_ 又称向后兼容模式，允许将传统网页渲染为作者意图。旧浏览器使用的非标准渲染规则。 不完整的、不正确的、缺少 DOCTYPE 声明或已知的 DOCTYPE 声明中普遍使用 2001 年以前的文件将在怪异模式中呈现。
+- _标准模式：_ 浏览器试图严格遵守 W3C 标准。新 HTML 网页有望被设计为符合标准的浏览器，这样做的结果就是，用现代 DOCTYPE 声明的页面将被用标准模式呈现。
```
Comment (yin1999) [inline files/zh-cn/learn/css/howto/css_faq/index.md:21]: remove extra space:

```suggestion
- *怪异模式*：又称向后兼容模式，允许将传统网页渲染为作者意图。旧浏览器使用的非标准渲染规则。不完整的、不正确的、缺少 DOCTYPE 声明或已知的 DOCTYPE 声明中普遍使用 2001 年以前的文件将在怪异模式中呈现。
- *标准模式*：浏览器试图严格遵守 W3C 标准。新 HTML 网页有望被设计为符合标准的浏览器，这样做的结果就是，用现代 DOCTYPE 声明的页面将被用标准模式呈现。
```

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+HTML 元素可以拥有一个 `id`/或 `class` 属性。 `id` 属性为元素指定应用一个有效名称，只能有一个具有该名称的元素。`class` 属性指定一个类名的元素，而这个名称可以被页面内的许多元素被使用。 CSS 允许你可以对特定的 `id` 和/或类名的元素应用样式。
+
+- 当你想要将样式规则应用于多个块和元素时，你应该使用 class 选择符。
+
+- 当你想给一个特定元素或块应用样式规则时就使用 ID 选择符。此样式将只用于与该特定 id 匹配的元素。
+
+较少样式的样式表通常性能更高。因此建议尽可能多地使用类， 保留 id 作为特定用途 (比如链接 label 标签和 form 元素或者为语义上唯一的元素应用样式)。
+
+> **Note:** 更深入层次的了解 [CSS 选择器](/zh-CN/docs/Learn/CSS/Building_blocks/Selectors)
```
Comment (yin1999) [inline files/zh-cn/learn/css/howto/css_faq/index.md:70]: https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN#%E5%A4%87%E6%B3%A8%E3%80%81%E8%AD%A6%E5%91%8A%E5%92%8C%E6%A0%87%E6%B3%A8

```suggestion
> **备注：** 更深入层次的了解 [CSS 选择器](/zh-CN/docs/Learn/CSS/Building_blocks/Selectors)。
```

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+## 为什么有效的 CSS 没有正确的渲染？
+
+浏览器使用 DOCTYPE 声明来选择是否使用更符合 Web 标准或兼容旧浏览器的 bug 的模式。在你的 HTML 的开始使用一个正确的和现代的 DOCTYPE 声明将改善浏览器标准执行。
+
+现代浏览器主要有两种渲染模式：
+
+- _怪异模式：_ 又称向后兼容模式，允许将传统网页渲染为作者意图。旧浏览器使用的非标准渲染规则。 不完整的、不正确的、缺少 DOCTYPE 声明或已知的 DOCTYPE 声明中普遍使用 2001 年以前的文件将在怪异模式中呈现。
+- _标准模式：_ 浏览器试图严格遵守 W3C 标准。新 HTML 网页有望被设计为符合标准的浏览器，这样做的结果就是，用现代 DOCTYPE 声明的页面将被用标准模式呈现。
+
+基于 Gecko 的浏览器， 有三分之一 _[Almost Standards Mode](/en-US/docs/Gecko's_%22Almost_Standards%22_Mode)_，只有一些小怪癖。
```
Comment (yin1999) [inline files/zh-cn/learn/css/howto/css_faq/index.md:23]: remove extra space: <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC>

```suggestion
基于 Gecko 的浏览器，有三分之一 _[Almost Standards Mode](/en-US/docs/Gecko's_%22Almost_Standards%22_Mode)_，只有一些小怪癖。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
Hi @jasonren0403, to keep the git history, you should spread the changes into two commits:

1. the first commmit is used to rename the file (using `git mv` command)
2. the second commit is used to content convertion and sync with english document

And I've given some suggestions below, please follow documents:

- <https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN>
- <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md>

to update markdown content

File: files/zh-cn/learn/css/howto/css_faq/index.md (hunk 3 above)
Comment (jasonren0403) [reply files/zh-cn/learn/css/howto/css_faq/index.md]: I have reorganized local commits, pleaes review updated contents.

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+slug: Learn/CSS/Howto/CSS_FAQ
+translation_of: Learn/CSS/Howto/CSS_FAQ
+tags:
+- CSS
+- Example
+- FAQ
+- Guide
+- Web
+- questions
+  original_slug: Web/CSS/Common_CSS_Questions
```
Comment (yin1999) [inline files/zh-cn/learn/css/howto/css_faq/index.md:12]:
```suggestion
original_slug: Web/CSS/Common_CSS_Questions
```

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 5]
```diff
@@ -2,183 +2,247 @@
...
 
-<h2 id="My_CSS_is_valid.2C_but_not_correctly_rendered">为什么我有效的 css 完全没有被渲染？ </h2>
+请尽可能地使用 HTML 5 DOCTYPE。
 
-<p>为了使浏览器渲染样式文件，CSS 样式表必须用 text/css 的 MIME。如果 Web 服务器不服务于这种类型，则 CSS 也不会被应用。</p>
+## 为什么有效的 CSS 完全没有被渲染？
 
-<h2 id="Difference_between_id_and_class">id 和 class 有什么不同？</h2>
+可能的原因如下：
+* 引入的 CSS 文件路径写错了。
```
Comment (yin1999) [inline files/zh-cn/learn/css/howto/css_faq/index.md:56]: 请空行并统一使用 `-` 来作为无序列表的前缀：

```suggestion
可能的原因如下：

- 引入的 CSS 文件路径写错了。
```

File: files/zh-cn/learn/css/howto/css_faq/index.md [hunk 6]
```diff
@@ -2,183 +2,247 @@
...
 
-<p>HTML元素可以拥有一个id/或class属性。 id属性为元素指定应用一个有效名称，只能有一个具有该名称的元素。class属性指定一个类名的元素，而这个名称可以被页面内的许多元素被使用。 CSS允许你可以对特定的id和/或类名的元素应用样式。<br>
- <br>
- 当你想给一个特定元素或块应用样式规则时就使用 ID 选择符。此样式将只用于与该特定 id 匹配的元素。</p>
+* 为了使浏览器渲染样式文件，CSS 样式表必须用 `text/css` 的 MIME。如果 Web 服务器没有使用这种类型处理文件，则 CSS 也不会被应用。
```
Comment (yin1999) [inline files/zh-cn/learn/css/howto/css_faq/index.md:58]:
```suggestion
- 为了使浏览器渲染样式文件，CSS 样式表必须用 `text/css` 的 MIME。如果 Web 服务器没有使用这种类型处理文件，则 CSS 也不会被应用。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
请修改，并合并commit（除了第一个rename commit）

**Comment by jasonren0403** [comment]
搞定了

**Pull request merged**